### PR TITLE
relabel 'What is Fixity?' as 'Introduction'

### LINKED ISSUE #360: Text Clean-up: Section 7 - Binary Resource Fixity
This issue is to update the text in Section 7 per the comments in the [this Google-Doc](https://docs.google.com/document/d/1n7ZVIRaiYG36Ye0AP4oSTYbDN915box6zVteXciojFc/edit?usp=sharing).

The updates should also include:
1. A `7.1 Introduction`
1. Replacement of "An Implementation" with "Implementations"... as applicable.

### PR DESCRIPTION
**Pull request by barmintor**
use "clients" instead of "a client" to parallel "implementations" language
fixes #360

### CODE REVIEW

**Review by escowles — DISMISSED** [review]

**Comment by zimeon** [comment]
Moving goalposts: "What is Fixity?" --> "Introduction" --> "" per #369

**Review by dannylamb — DISMISSED** [review]

**Review by zimeon — CHANGES_REQUESTED** [review]
Needs "Introduction" title removed

File: index.html [hunk 1]
```diff
@@ -1347,12 +1347,12 @@ <h4>A basic notification with some additional detail</h4>
 
     <section id="binary-fixity">
       <h2>Binary Resource Fixity</h2>
-      <section id="what-is-fixity">
-        <h2>What is fixity?</h2>
+      <section id="binary-fixity-introduction">
+        <h2>Introduction</h2>
```
Comment (awoods) [inline index.html]: I believe the latest agreement is to not include the `Introduction` header.

**Review by awoods — CHANGES_REQUESTED** [review]

**Comment by barmintor** [comment]
well that was a headache far out of proportion with the change, silly forks.

**Comment by dannylamb** [comment]
I thought we were dropping the intro header altogether? I would axe the "What is Fixity?" line.
```html
    <section id="binary-fixity">
      <h2>Binary Resource Fixity</h2>
      <section id="fixity-introduction">
        <blockquote class="informative">
<p>
```

**Comment by barmintor** [comment]
@dannylamb you're absolutely right

**Review by dannylamb — DISMISSED** [review]

File: index.html [hunk 2]
```diff
@@ -1362,9 +1362,7 @@ <h4>A basic notification with some additional detail</h4>
     </section>
 
     <section id="binary-fixity">
-      <h2>Binary Resource Fixity</h2>
```
Comment (zimeon) [inline index.html]: We still need this major heading I think

Comment (barmintor) [reply index.html]: what would Darth Vader say here

Comment (dannylamb) [reply index.html]: @zimeon D'oh... yes, we should still have that.

Comment (barmintor) [reply index.html]: ok, I have restored the h2 and marked the other two headers in the section up as h3

**Review by zimeon — APPROVED** [review]

**Review by escowles — APPROVED** [review]

**Review by awoods — APPROVED** [review]

**Pull request merged**